package.json

```diff
@@ -40,7 +40,10 @@
         "build": "turbo run build",
         "ci:build": "turbo run build --cache-dir=\".turbo\"",
         "test": "vitest run --silent",
-        "test:e2e": "node test/e2e/run.mjs",
+        "test:e2e": "npm run test:e2e:scrapers && npm run test:e2e:sdk",
+        "test:e2e:scrapers": "node test/e2e/runScraperTests.mjs",
+        "test:e2e:sdk": "npm run test:e2e:sdk:tarball && node test/e2e/runSdkTests.mjs",
+        "test:e2e:sdk:tarball": "cd packages/apify; mv $(npm pack | tail -n 1) ../../test/e2e/apify.tgz",
```

Comment:
do you really want `;` here and not `&&`?

also, how exactly does this work?

Reply:
It's very unlikely that the `cd` would fail, so... `&&` is correct, but it's bordeline bikeshedding.

What happens is that `test:e2e:sdk:tarball` builds an npm-install-able .tgz file in the `test/e2e` directory. Then it is copied into each Actor build and picked up and installed in the Dockerfile.

Reply:
Well, I had to look up what `;` does, unlike `&&` which we all use pretty much everywhere. That's what triggered this comment mainly, and when I read what `;` does, it just feels wrong to use it. Agreed its very unlikely to cause anything else then confusion, but that's also a problem.

> What happens is that test:e2e:sdk:tarball builds an npm-install-able .tgz file in the test/e2e directory. Then it is copied into each Actor build and picked up and installed in the Dockerfile.

We should probably adopt this in the crawlee e2e tests too, that weird script for copying packages haunts me every time I deal with the e2e tests there.